### PR TITLE
Cult pylons heal at the same rate again

### PR DESCRIPTION
**Pull request by Archanial**
## About The Pull Request

Delta time is 0.2, they healed 1 before.
Also small refactor to the code

closes: #4814

## Changelog
:cl:
fix: Cult pylons heal 1 damage per tick again.
/:cl:

### CODE REVIEW

**Comment by Cenrus** [comment]
Wasn't healing 1 damage per tick when I tested on local, it's abysmally slow. 0.5 * 0.2 is 0.1

**Comment by BrumalLake** [comment]
yeah he didnt actually change any numbers just a refactor

**Comment by PowerfulBacon** [comment]
is anything even changed?

**Comment by Archanial** [comment]
> 
> 
> Wasn't healing 1 damage per tick when I tested on local, it's abysmally slow. 0.5 * 0.2 is 0.1

forgot to push

File: code/modules/antagonists/cult/cult_structures.dm [hunk 1]
```diff
@@ -172,19 +172,20 @@
...
+			new /obj/effect/temp_visual/heal(get_turf(src), "#960000")
+			if(ishuman(L))
+				L.adjustBruteLoss(-5*delta_time, 0)
+				L.adjustFireLoss(-5*delta_time, 0)
+				L.updatehealth()
+				if(L.blood_volume < BLOOD_VOLUME_NORMAL)
 					L.blood_volume += 1.0
+			else if(isshade(L) || isconstruct(L))
+				var/mob/living/simple_animal/M = L
+				M.adjustHealth(-1.5*delta_time)
```
Comment (jupyterkat) [inline code/modules/antagonists/cult/cult_structures.dm]: change this too

Comment (Archanial) [reply code/modules/antagonists/cult/cult_structures.dm]:
```suggestion
				M.adjustHealth(-15*delta_time)
```

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**